util/reggen/field.py

```diff
@@ -39,6 +41,10 @@
     'tags': [
         's',
         "tags for the field, followed by the format 'tag_name:item1:item2...'"
+    ],
+    'mubi': [
+        'b',
+        "boolean flag for whether the field is a multi-bit type"
```

Comment:
Maybe "a multi-bit boolean"? I don't see it being used for other than true/false?